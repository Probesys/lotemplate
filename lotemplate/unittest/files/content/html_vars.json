{
  "foo": {
    "type": "html",
    "value": "simple puis <strong>texte en gras</strong>"
  },
  "bar": {
    "type": "text",
    "value": "simple puis <strong>texte en gras</strong>"
  }
}