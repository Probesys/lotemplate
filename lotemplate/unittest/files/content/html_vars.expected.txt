
Var html :

 simple puis texte en gras


var text

simple puis <strong>texte en gras</strong>


